src/code.sh

```
@@ -375,9 +375,13 @@ main() {
        find subjob_output -type f -name "*.FusionInspector.fusions.abridged.tsv.coding_effect" -print0 | xargs -0 -I {} mv {} ./coding_effect_files
        python3 merge_fusions_tsv.py -a coding_effect_files/*.coding_effect -o combined_files
 
-       ls -l combined_files
-
-       _create_fusion_inspector_report
+       # if there is fusions predicted. Create a html, else do not create a html as it'll be empty and fail
+       number_of_fusions=$(tail -n +2 combined_files/${prefix}.FusionInspector.fusions.abridged.coding_effect.merged.tsv | wc -l)
+       if [ $number_of_fusions >= 1 ]; then
```

Shellcheck (0.10.0)

[error] >= is not a valid operator. Use -ge .

(SC2071)